test/functional/helpers/spectrum.js

```diff
@@ -81,6 +81,22 @@ module.exports = {
       selector,
       expectError: createExpectError(selector),
       expectValue: createExpectValue(selector),
+      async expectSelectedOptionLabel(t, label) {
+        await switchToIframe(t);
+        await t
+          .expect(selector.find(".spectrum-Dropdown-label").innerText)
+          .eql(label);
+      },
+      async expectOptionLabels(t, labels) {
+        await switchToIframe(t);
+        await t.click(selector.find("button"));
+        const optionLabels = popoverSelector.find(".spectrum-Menu-itemLabel");
+        for (let i = 0; i < labels.length; i += 1) {
+          // eslint-disable-next-line no-await-in-loop
```

Comment:
I get why this is a rule (https://eslint.org/docs/rules/no-await-in-loop), but running the `t.expect(optionLabels.nth(i).withText(labels[i]).exists).ok();` code for each label in parallel isn't actually effective with TestCafe because TestCafe runs them sequentially anyway (it maintains a queue internally for all `t.*` calls) and, as such, would only serve to make this code more complicated.